src/screens/ChatInfoScreen/ChatInfoScreen.tsx

```diff
@@ -24,6 +24,7 @@ import BlockComponent from '../../components/BlockComponent';
 import ChannelImage from '../../components/ChatList/elements/ChannelImage';
 import ModalAction from '../GroupInfo/elements/ModalAction';
 import ModalActionAnonymous from '../GroupInfo/elements/ModalActionAnonymous';
+import UserSchema from '../../database/schema/UserSchema';
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The import of `UserSchema` is a new addition and seems to be used correctly in the context of this file. It's being used as a type for the `item` parameter in the `getUsername` function.
```diff
+ import UserSchema from '../../database/schema/UserSchema';
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->